js/radio.js

```diff
@@ -59,13 +59,20 @@
 		constructor: Radio,
 
 		destroy: function () {
-			this.$parent.remove();
 			// remove any external bindings
 			// [none]
 			// empty elements to return to original markup
 			// [none]
 			// return string of markup
-			return this.$parent[0].outerHTML;
+			if (this.$parent) {
```

Comment:
oh i see it now. certainly could be adjusted. or add a `this.isWrapped` attribute to be initialized with `true` and set to false with 41-43